ReactiveCocoa/Swift/Flatten.swift

```diff
@@ -429,13 +429,11 @@ private final class ConcatState<Value, Error: Swift.Error> {
 			return
 		}
 
-		var shouldStart = true
-
-		queuedSignalProducers.modify { queue in
+		let shouldStart: Bool = queuedSignalProducers.modify { queue in
 			// An empty queue means the concat is idle, ready & waiting to start
 			// the next producer.
-			shouldStart = queue.isEmpty
-			queue.append(producer)
+			defer { queue.append(producer) }
```

Comment:
Shorthand arguments aren't used in any closures with `defer`, which somehow prohibits the use.